src/page.js

```diff
@@ -202,10 +202,15 @@ export default function wikiPage(rawPageInfo, apiOptions) {
 	 * @example
 	 * new Wiki().page('Batman').then(page => page.info('alter_ego'));
 	 * @method WikiPage#info
-	 * @param  {String} [key] - Information key
+	 * @param  {String} [key] - Information key. Falsy keys are ignored
+   * @param  {Object} [options] - Infobox parser options
 	 * @return {Promise} - info Object contains key/value pairs of infobox data, or specific value if key given
 	 */
-	function info(key) {
+	function info(key, options) {
+    if (key && typeof key === 'object') {
```

Comment:
The solution allows both ways of setting the options. I would defend letting the users choose where they want/need to set up the options. But you are the owner. I can change it if you want to, no prob.